src/Ilios/CoreBundle/Entity/LearningMaterial.php

```diff
@@ -296,13 +293,12 @@ public function getToken()
      */
     public function generateToken()
     {
-        $generator = new SecureRandom();
-        $random = $generator->nextBytes(128);
-        
+        $random = random_bytes(128);
```

Comment:
random_bytes seems to be a PHP 7 function - needing a polyfill to work with 5.5+
https://github.com/paragonie/random_compat

Since it doesn't look like composer.json was updated I'm guessing that another library we use is including that for us, but we should probably explicitly declare it in case that changes.